scripts/e2e-express-embedded-sign-in-widget.sh

```diff
@@ -19,8 +19,6 @@ export ORG_OIE_ENABLED=true
 export USERNAME=mary@acme.com
 get_secret prod/okta-sdk-vars/password PASSWORD
 get_vault_secret_key devex/auth-js-sdk-vars a18n_api_key A18N_API_KEY
-export FB_USERNAME=js_ekdtypn_user@tfbnw.net
```

Comment:
We have disabled FB login tests, instead we use okta as 3rd party OIDC provider in tests